### PR TITLE
ab: instructor message create account

### LINKED ISSUE #476: Instructor Account: change message at top when user sets up Instructor Account
The message at the top of the "Create Instructor Account' should be changed to reflect that the user is not trying to set up a student account. The message should read:

In the instructor account, instructors set up and customize their own StarCellBio assignments and experiments. The email address for the instructor account must be different from those used to set up other StarCellBio accounts. To set up an instructor account, enter your email and password below and then click Create Account. 

[Please note that the 'Create Account' text in the message above should be in bold and in the same green color as the button.]

### PR DESCRIPTION
**Pull request by annagav**
Fix #476

### CODE REVIEW

**Comment by noisecapella** [comment]
By default the student is selected in the sign up form, but you don't see the student message unless you explictly click the student radio button. Can you update this to also display the student message by default, if the student radio button is selected?

File: auth/templates/uniform/allauth/account/signup.html [hunk 1]
```diff
@@ -143,15 +141,28 @@
...
+            "up other StarCellBio accounts. To set up an instructor account, enter " +
+            "your email and password below and then click Create Account.";
     document.getElementById('div_id_course_code').style.display = 'none';
+    document.getElementById('toggle_message').innerHTML = message;
 }
 
-function showCourseCode(){
+function showStudentForm(){
+    var message = 'Your instructor should have provided you with a code that you need ' +
+            'to enter below to register and use this StarCellBio tool.Please contact ' +
```
Comment (noisecapella) [inline auth/templates/uniform/allauth/account/signup.html]: Can you add a space at the end of the sentence at `... StarCellBio tool.Please contact`

**Comment by annagav** [comment]
Fixed!

**Comment by noisecapella** [comment]
:+1:

**Pull request merged**